tests/runtests.py

```diff
@@ -248,6 +248,8 @@ def _module_match_label(module_label, label):
 
     apps.set_installed_apps(settings.INSTALLED_APPS)
 
+    os.environ['DJANGO_TEST_SUITE'] = "1"
```

Comment:
single quotes + `RUNNING_DJANGOS_TEST_SUITE` could be more descriptive. Also comment `# Set an environment variable that other code may consult to see if Django's own test suite is running.`